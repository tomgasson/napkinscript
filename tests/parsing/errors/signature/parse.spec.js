runParser(__dirname, true, true);
