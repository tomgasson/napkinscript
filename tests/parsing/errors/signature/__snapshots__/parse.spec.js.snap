// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`closingBraces.nsi 1`] = `
"=====Parsetree==========================================
val x : int
val y : int
=====Errors=============================================

File \\"/napkinscript/tests/parsing/errors/signature/closingBraces.nsi\\", line 3, characters 0-1:


1 â”‚  let x: int
2 â”‚  
[31m3[0m â”‚  [31m}[0m}}
4 â”‚  
5 â”‚  let y: int

I'm not sure what to parse here when looking at \\"}\\".


File \\"/napkinscript/tests/parsing/errors/signature/closingBraces.nsi\\", line 3, characters 1-2:


1 â”‚  let x: int
2 â”‚  
[31m3[0m â”‚  }[31m}[0m}
4 â”‚  
5 â”‚  let y: int

I'm not sure what to parse here when looking at \\"}\\".


File \\"/napkinscript/tests/parsing/errors/signature/closingBraces.nsi\\", line 3, characters 2-3:


1 â”‚  let x: int
2 â”‚  
[31m3[0m â”‚  }}[31m}[0m
4 â”‚  
5 â”‚  let y: int

I'm not sure what to parse here when looking at \\"}\\".



========================================================"
`;
